exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -66,14 +66,26 @@ public int GetTotalPlayersCount()
             return totalPlayers.Count;
         }
 
+        private static bool IsRollResultOdd(int rollResult)
+        {
+            if (rollResult % 2 != 0)
+            {
+                return true;
+            }
+            else
+            {
+                return false;
+            }
+        }
+
         public void RollDice(int rollResult)
         {
             Console.WriteLine(totalPlayers[currentPlayer] + " is the current player");
             Console.WriteLine("They have rolled a " + rollResult);
 
             if (playerPenaltyBoxState[currentPlayer])
             {
-                if (rollResult % 2 != 0)
+                if (IsRollResultOdd(rollResult))
```

Comment:
I can see why you extracted this into a method, as it does make it easier to understand what the conditional is checking. But I feel it may add a bit more noise to your code than needed.